testing/src/main/java/org/oppia/testing/FakeEventLogger.kt

```diff
@@ -0,0 +1,77 @@
+package org.oppia.testing
+
+import android.content.Context
+import android.os.Bundle
+import org.oppia.app.model.EventLog
+import org.oppia.util.logging.EventBundleCreator
+import org.oppia.util.logging.EventLogger
+import org.oppia.util.logging.firebase.EXPLORATION_ID_KEY
+import org.oppia.util.logging.firebase.PRIORITY_KEY
+import org.oppia.util.logging.firebase.QUESTION_ID_KEY
+import org.oppia.util.logging.firebase.STORY_ID_KEY
+import org.oppia.util.logging.firebase.TIMESTAMP_KEY
+import org.oppia.util.logging.firebase.TOPIC_ID_KEY
+import java.util.*
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**  A test specific fake for the event logger. */
+@Singleton
+class FakeEventLogger @Inject constructor() : EventLogger, EventBundleCreator {
+  private val eventList = ArrayList<EventLog>()
+  private var bundle = Bundle()
+
+  override fun logEvent(context: Context, eventLog: EventLog) {
+    eventList.add(eventLog)
+    bundle =
+      when (eventLog.context.activityContextCase) {
+        EventLog.Context.ActivityContextCase.EXPLORATION_CONTEXT ->
+          createExplorationContextBundle(eventLog)
+        EventLog.Context.ActivityContextCase.QUESTION_CONTEXT ->
+          createQuestionContextBundle(eventLog)
+        else -> defaultBundle(eventLog)
+      }
```

Comment:
What is the purpose of this bundle exactly?

Reply:
It's basically a default value, but yes if we implement the approach in my earlier comment, we'll not need it as there will always be a context. 
This was basically made to log events where all the info such as topicId, storyId, etc. wasn't present. For eg - Home. 
However, we can accommodate a `logEventWithNoContext` as well. Thoughts?

Reply:
Yes we would need a logEventWithNoContext as well

Reply:
Done, PTAL :)

Reply:
We are still keeping the bundle though? What is the purpose of it?

Reply:
Modified, PTAL.